test/torch/mpc/test_fss.py

```diff
@@ -1,58 +1,64 @@
+import pytest
 import syft
 import torch as th
 
 from syft.frameworks.torch.mpc.fss import DPF, DIF, n
 
 
-def test_DPF():
+@pytest.mark.parametrize("op", ["eq", "le"])
+def test_fss_klass(op):
+    klass = {"eq": DPF, "le": DIF}[op]
```

Comment:
The PEP8 standard for avoiding name collisions with reserved words like `class` to append an underscore like `class_`.